examples/js/controls/OrbitControls.js

```diff
@@ -933,7 +933,15 @@ THREE.OrbitControls = function ( object, domElement ) {
 	scope.domElement.addEventListener( 'touchend', onTouchEnd, false );
 	scope.domElement.addEventListener( 'touchmove', onTouchMove, false );
 
-	window.addEventListener( 'keydown', onKeyDown, false );
+	scope.domElement.addEventListener( 'keydown', onKeyDown, false );
+
+	// make sure element can receive keys.
+
+	if ( scope.domElement !== document && scope.domElement.tabIndex === -1) {
```

Comment:
scope.domElement.tabIndex could be any negative number
https://www.w3.org/TR/html51/editing.html#the-tabindex-attribute

Reply:
Not sure what your point is. I'm only checking if it's the default value which is the only way to *guess* if the user didn't set it. If they did set it then I don't want to change it.

>  Its default value is 0 for elements that are focusable and -1 for elements that are not focusable.